tests/garage/torch/algos/test_loss_function_utils.py

```diff
@@ -0,0 +1,38 @@
+import numpy as np
+import tensorflow as tf
+import torch
+
+import garage.tf.misc.tensor_utils as tf_utils
+import garage.torch.algos.loss_function_utils as torch_loss_utils
+from tests.fixtures import TfGraphTestCase
+
+
+class TestLossFunctionUtils(TfGraphTestCase):
+
+    def test_compute_advantages(self):
+        discount = 0.99
+        gae_lambda = 1.
+        rewards_val, baselines_val = np.ones((4, 6)), np.zeros((4, 6))
```

Comment:
please use a non-zero value for the baselines. otherwise, an entire component of this computation is inactive.

Reply:
here are some ideas for non-trivial sequences which are not-random:
`np.arange`, the fibonacci sequence, digits of `np.pi`, digits of e, etc. to get non-huge numbers you can divide many of these by some constant multiple.

Reply:
fixed